country-a-service/epps-service/src/main/java/dk/nsp/epps/service/PrescriptionService.java

```diff
@@ -110,4 +113,25 @@ public void submitDispensation(@NonNull String patientId, @NonNull Document disp
             throw new CountryAException(HttpStatus.INTERNAL_SERVER_ERROR, "CreatePharmacyEffectuation failed", e);
         }
     }
+
+    public UndoEffectuationResponseType undoDispensation(@NonNull String patientId, Document cdaToDiscard) throws JAXBException, MapperException, XPathExpressionException {
+        String cpr = PatientIdMapper.toCpr(patientId);
+        final var request = GetPrescriptionRequestType.builder()
+            .withPersonIdentifier().withSource("CPR").withValue(cpr).end()
+            .withIncludeOpenPrescriptions().end()
+            .withIncludeEffectuations(true)
+            .build();
+        log.debug("Looking up info for {}", cpr);
```

Comment:
I suppose you make a good point. Since we're on debug anyway, we can assume control enough over the flow to figure out which call it is connected to anyways, because otherwise it would hardly give more information :). I don't think we need any logging here for info+